packages/volto/src/components/theme/View/__snapshots__/View.test.jsx.snap

```diff
@@ -7,9 +7,15 @@ exports[`View renders a document view 1`] = `
   <div
     id="ContentMetadataTags"
   />
+  <div
+    id="SlotRenderer"
+  />
```

Comment:
Is there anyway we can actually test things are rendered here rather than just seeing the render wrapper? Seems like a false sense of security with the tests seeing that things 'pass' but rendering could be completely broken.

Reply:
For the `View` it's better like this. We have a Cypress test for it, so we are "safe" I will add a unit test also for the `SlotRenderer`.

Reply:
Done.